hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -106,10 +104,11 @@ protected Object run() throws Exception {
 
       @Override
       protected Object getFallback() {
-        if (fallback == null) {
+        if (fallbackFactory == null) {
           return super.getFallback();
         }
         try {
+          Object fallback = fallbackFactory.create(getFailedExecutionException());
```

Comment:
note: if all we want to do is log the cause, we can delete the entire pull request and just log here!

```java
Throwable cause = getFailedExecutionException();
 if (logger.isLoggable(Level.FINE)) {
  logger.log(Level.FINE, "fallback due to: " + cause.getMessage(), cause);
}
```